### PR TITLE
Run tests which were ignored in janusgraph-backend-testutils

### PR DESCRIPTION
**Pull request by porunov**
Signed-off-by: Oleksandr Porunov <alexandr.porunov@gmail.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

File: janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java [hunk 1]
```diff
@@ -19,85 +19,91 @@
...
-        map.put(STORAGE_BACKEND.toStringWithoutRoot(), "inmemory");
+        final Map<String, Object> map = getStorageConfiguration();
         map.put(UNIQUE_INSTANCE_ID_HOSTNAME.toStringWithoutRoot(), true);
         final MapConfiguration config = ConfigurationUtil.loadMapConfiguration(map);
         final StandardJanusGraph graph = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
-        assertEquals(graph.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName());
-				graph.close();
+        assertEquals(1, graph.openManagement().getOpenInstances().size());
+        assertEquals(toCurrentInstance(Inet4Address.getLocalHost().getHostName()), graph.openManagement().getOpenInstances().iterator().next());
```
Comment (li-boxuan) [inline janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java:90]: This is failing locally for me. Do you know why?
```
Expected :liboxuans-MacBook-Pro.local(current)
Actual   :liboxuans-MacBook-Pro-local(current)
```

Comment (porunov) [reply janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java:90]: Hmm interesting. It works for me locally. Looks it changed `.` to `-` for some reason

Comment (porunov) [reply janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java:90]: Okay, I see the reason. Those characters are replaced here: `UniqueInstanceIdRetriever.computeUniqueInstanceId`

Comment (porunov) [reply janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java:91]: @li-boxuan could you try now?

Comment (li-boxuan) [reply janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java:91]: Nice, they are working now

**Review by li-boxuan — APPROVED** [review]
LGTM 👍  I can see InmemoryGraphDatabaseConfigurationInstanceIdTest is executed in GitHub Actions.

**Pull request merged**